types/options.d.ts

```diff
@@ -32,6 +32,10 @@ export interface CoreAnnotationOptions extends AnnotationEvents {
   value?: Scriptable<number|string, PartialEventContext>,
   xScaleID?: Scriptable<string, PartialEventContext>,
   yScaleID?: Scriptable<string, PartialEventContext>,
+  shadowColor: Scriptable<Color, PartialEventContext>,
+  shadowBlur: Scriptable<number, PartialEventContext>,
+  shadowOffsetX: Scriptable<number, PartialEventContext>,
+  shadowOffsetY: Scriptable<number, PartialEventContext>
```

Comment:
Missing the `?` behind the property name to make them optional

Reply:
:+1: done!